scripts/increment-cargo-version.sh

```diff
@@ -31,7 +31,7 @@ readCargoVariable() {
 }
 
 # shellcheck disable=2207
-Cargo_tomls=($(find . -name Cargo.toml -not -path './web3.js/examples/*'))
+Cargo_tomls=($(find . -mindepth 2 -name Cargo.toml -not -path './web3.js/examples/*'))
```

Comment:
@mvines I just went hunting for this change to investigate why I suddenly needed to make it today and realized I accidentally committed it.  I was getting kicked out because it was trying to pull the current version out of the workspace Cargo.toml, which of course doesn't have it.  I'm still not sure what the root cause is, but I don't think the change is wrong.  WDYT about sticking it in master too?

Reply:
Sure that looks ok to me